shared/common-adapters/list.desktop.js

```diff
@@ -16,7 +17,47 @@ class List extends PureComponent<Props<any>, void> {
       return null
     }
     const item = this.props.items[index]
-    const children = this.props.renderItem(index, item)
+    let children
+
+    // If we're in dev, let's check our estimates and warn if we're using margins
+    if (__DEV__ && !!this.props.itemSizeEstimator) {
```

Comment:
I realize this adds an extra div that might affect the layout of things, although I'm not sure exactly when it would affect it. It is pretty nice to have this warning instead of having to remember to check these estimates manually when you update code.

Reply:
ideally we could do this w/o this change in hierarchy. (ideally we'd move away from react-list to react-window anyways)
maybe we can just get a ref and iterate over the elements and get the bounding rects and compare to the list vs adding measure